### PR TITLE
Fix/GUM 64 add placeholders on empty list

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: src/screens/opportunities/opportunities.tsx [hunk 1]
```diff
@@ -49,6 +49,9 @@ const OpportunitiesScreen: React.FC = () => {
           </View>
         )}
       />
+      {Boolean(!opportunities?.length) && (
```
Comment (liza-veis) [inline src/screens/opportunities/opportunities.tsx]: This expression will always return boolean
```suggestion
      {!opportunities?.length && (
```

File: src/screens/profile/profile.tsx [hunk 2]
```diff
@@ -164,6 +164,9 @@ const ProfileScreen: React.FC = () => {
                     />
                   </View>
                 ))}
+                {Boolean(!careerExperience?.length) && (
```
Comment (liza-veis) [inline src/screens/profile/profile.tsx]:
```suggestion
                {!careerExperience?.length && (
```

File: src/components/nothing-here/nothing-here.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { HeadingLevel } from 'src/common/enums';
+import { Heading } from '..';
+import useStyles from './styles';
+
+type NothingHereProps = {
+  children: string;
+};
+
+const NothingHere: React.FC<NothingHereProps> = ({ children: text }) => {
```
Comment (liza-veis) [inline src/components/nothing-here/nothing-here.tsx]: Maybe it's better to name this component like EmptyMessage?

Comment (R-Mielamud) [reply src/components/nothing-here/nothing-here.tsx]: EmptyListMessage

**Review by liza-veis — APPROVED** [review]

**Review by yevhenprybytok — APPROVED** [review]

**Pull request merged**